lms/djangoapps/grades/models.py

```diff
@@ -149,12 +129,11 @@ class VisibleBlocks(models.Model):
     in the blocks_json field. A hash of this json array is used for lookup
     purposes.
     """
-    CACHE_NAMESPACE = u"grades.models.VisibleBlocks"
     blocks_json = models.TextField()
     hashed = models.CharField(max_length=100, unique=True)
     course_id = CourseKeyField(blank=False, max_length=255, db_index=True)
 
-    objects = VisibleBlocksQuerySet.as_manager()
+    _CACHE_NAMESPACE = u"grades.models.VisibleBlocks"
```

Comment:
**Note to reviewers:** Moved `_CACHE_NAMESPACE` down for consistency with other models.  Also, added an underscore throughout to be clear that it's an internal field.